src/game/game_player_record.py

```diff
@@ -7,7 +7,14 @@ class GamePlayerRecord(polymodel.PolyModel):
     player_key = ndb.KeyProperty(kind="User", required=True)
     player_placement = ndb.IntegerProperty(required=True,
                                            choices=[1, 2, 3, 4, 5])
-    rank_points_earned = ndb.IntegerProperty()
+    rank_points_earned = ndb.FloatProperty(default=0.0)
```

Comment:
@josephmcgovern-wf you could certainly round this when displaying, but in order for Trueskill to work with it, it needs to be a float


Also, it should be noted that once we get past the first 10 games and elasticity goes down, your skill rating usually only fluctuates by like .2 on a win.